tests/release/pipelines/fbc_release.go

```diff
@@ -15,186 +15,272 @@ import (
+}
+
+func assertReleaseCRSucceeded(devFw framework.Framework, devNamespace, managedNamespace, fbcAppName string, component *appservice.Component) {
+	Eventually(func() error {
+		buildPr, err := devFw.AsKubeDeveloper.HasController.GetComponentPipelineRun(component.Name, fbcAppName, devNamespace, "")
+		Expect(err).ShouldNot(HaveOccurred())
+		snapshot, err := devFw.AsKubeDeveloper.IntegrationController.GetSnapshot("", buildPr.Name , "", devNamespace)
+		Expect(err).ShouldNot(HaveOccurred())
+		releaseCR, err := devFw.AsKubeDeveloper.ReleaseController.GetRelease("", snapshot.Name, devNamespace)
+		if err != nil {
+			return err
+		}
+		GinkgoWriter.Println("Release CR: ", releaseCR.Name)
+		if !releaseCR.IsReleased() {
+			return fmt.Errorf("release %s/%s is not marked as finished yet", releaseCR.GetNamespace(), releaseCR.GetName())
+		}
+		return nil
+	}, releaseCreationTimeout, defaultInterval).Should(Succeed())
+}
+
+func CreateComponent(devFw framework.Framework, devNamespace, managedNamespace, fbcAppName, fbcCompName string) *appservice.Component {
+	// using cdq since git ref is not known
+	var componentDetected appservice.ComponentDetectionDescription
+	cdq, err := devFw.AsKubeAdmin.HasController.CreateComponentDetectionQuery(fbcCompName, devNamespace, fbcSourceGitUrl, "", "", "", false)
+	Expect(err).NotTo(HaveOccurred())
+	Expect(cdq.Status.ComponentDetected).To(HaveLen(1), "Expected length of the detected Components was not 1")
+
+	for _, compDetected := range cdq.Status.ComponentDetected {
+		componentDetected = compDetected
+	}
+
+	component, err := devFw.AsKubeAdmin.HasController.CreateComponent(componentDetected.ComponentStub, devNamespace, "", "", fbcAppName, false, map[string]string{})
+	Expect(err).NotTo(HaveOccurred())
+	GinkgoWriter.Println("component : ", component.Name)
+	Expect(err).ShouldNot(HaveOccurred())
+	return component
+}
+
+func CreateEnterpriseContractPolicy(fbcECPName string,managedFw framework.Framework, devNamespace, managedNamespace string) {
+	defaultEcPolicySpec := ecp.EnterpriseContractPolicySpec{
+		Description: "Red Hat's enterprise requirements",
+		PublicKey:   "k8s://openshift-pipelines/public-key",
+		Sources: []ecp.Source{{
+			Name:   "Default",
+			Policy: []string{ecPolicyLibPath, ecPolicyReleasePath},
+			Data:   []string{ecPolicyDataBundle, ecPolicyDataPath},
+		}},
+		Configuration: &ecp.EnterpriseContractPolicyConfiguration{
+			Exclude:     []string{"cve", "step_image_registries", "tasks.required_tasks_found:prefetch-dependencies"},
+			Include:     []string{"minimal"},
+		},
+	}
+
+	_, err := managedFw.AsKubeDeveloper.TektonController.CreateEnterpriseContractPolicy(fbcECPName, managedNamespace, defaultEcPolicySpec)
+	Expect(err).NotTo(HaveOccurred())
+
+}
+
+func CreateReleasePlanAdmission(fbcRPAName string, managedFw framework.Framework, devNamespace, managedNamespace, fbcAppName, fbcECPName, pathInRepoValue, hotfix, issueId string) {
```

Comment:
don't we have a util for this? The only unique thing is the data map which we could pass as an argument, no?

Reply:
Maybe you should not export the function then to make it more clear (use lowercase c instead of capital C as the first character in the function name)?

Reply:
Updated, thanks!

Reply:
I still see it as `CreateFBCReleasePlanAdmission` instead of `createFBCReleasePlanAdmission`

Reply:
Updated